test/testExplorer/TestExplorer.ts

```diff
@@ -1,6 +1,6 @@
 import "../../less/hostedexplorer.less";
 import { TestExplorerParams } from "./TestExplorerParams";
-import { DatabaseAccountsGetResponse } from "@azure/arm-cosmosdb/esm/models";
+import {​​​​​​​​ Capability, DatabaseAccount }​​​​​​​​ from"../../src/Contracts/DataModels";
```

Comment:
I saw Srinath also sent a PR with the chnages in this file. Please sync with him if he is checking in or merge the conflicts  appropriately.

Reply:
I will merge that PR, and chris can resolve conflicts. working with steve to fix the build first